### PR TITLE
148 reset form

### LINKED ISSUE #148: Clear Form -> Reset Form
**As a** *curator*
**I want** *to rather than clearing the form on the substance page, to reset the form back to its original state when the page was loaded*
**So that** *i can easily undo the changes that I may have made to the chemical and see the original*

**Acceptance Criteria:**
Rename the "Clear" button to "Reset" button
Rather than emptying the form, clicking the reset button returns the substance and compound to the original state that existed before any edits.

**Test(s):**

### PR DESCRIPTION
**Pull request by hausman-gdit**
Closes #148 

Adds ability to reset changes to the Substance Form and Chemical Editor Window.

- There are two buttons; one for the Substance Form and one for the Chemical Editor Window.
- Test for Substance Form reset
- Tests for Marvin & Ketcher window reset

**Substance**
148-reset-substance
**Compound**
148-reset-compound

### CODE REVIEW

File: tests/e2e/specs/substance.js [hunk 1]
```diff
@@ -697,6 +703,81 @@ describe("The substance page authenticated access", () => {
       .its("request.body.data.relationships.queryStructureType.data.id")
       .should("contain", "markush-query");
   });
+  it("should reset changes to the Ketcher window", () => {
```
Comment (michael-barbour) [inline tests/e2e/specs/substance.js:706]: This test seems great and passes.  Do you think another one that tests functionality on Marvin would be good?

**Review by michael-barbour — CHANGES_REQUESTED** [review]
I'm having an issue with the Ketcher functionality when no substance is loaded.

**Steps to Reproduce:**
 - Navigate to `/substance`
 - Make edits to ketcher frame (draw H2O)
 - Click "Reset substance"

**Expected:**
 - The KetcherWindow is returned to being blank.  Reset and Save compound buttons are disabled

**Actual:**
 - Nothing happens

This appears to be working as expected on MarvinWindow for Ill Defined Compounds.

File: tests/e2e/specs/substance.js (hunk 1 above)
Comment (hausman-gdit) [reply tests/e2e/specs/substance.js:706]: We had talked a little bit about using "Export" on the Marvin window to check what compound is drawn (or not drawn) and you'd mentioned it was pretty sketchy for Cypress to access the export window contents. 

The Marvin test (ln 754) currently adds Cyclohexane to the window then resets. The 'pass' condition is that the "Save Compound" & "Reset Compound" buttons are disabled after reset. I'm not sure what else I could do to check the contents of the window aside from the "Export" function.

Comment (michael-barbour) [reply tests/e2e/specs/substance.js:706]: You know I completely missed the following test `it("should reset changes to the Marvin window", () => { })`.  That test was exactly what I am looking for.  

Sorry for the confusion.

**Review by michael-barbour — APPROVED** [review]
Tests pass and functionality looks great.

**Pull request merged**